src/validations/actionSchema.ts

```diff
@@ -21,3 +21,9 @@ export const lifecycleIdSchema = Yup.object().shape({
   nextSteps: Yup.string(),
   feedback: Yup.string().required('Please fill out email')
 });
+
+export const rejectIntakeSchema = Yup.object().shape({
+  nextSteps: Yup.string(),
```

Comment:
Safe to omit if we aren't validating it.

Reply:
I think this is actually required for this action. Thanks for the catch.